### ISSUE TITLE
Credential Rotation Version 5.0.1

### ISSUE THREAD

**Issue by hunttom** [99designs/aws-vault#485]
Unable to rotate credentials on MacPro Catalina.
AWS-Vault installed with HomeBrew.
v.5.0.1

## Error
Tried credential rotation. Getting consistent response:
```
>> aws-vault rotate X
aws-vault: error: InvalidClientTokenId: The security token included in the request is invalid
	status code: 403, request id: 20b88003-3f1d-42dd-ae8d-7ea44d649ec1
```
Also AWSCLI/Boto/Python Versions:
```
aws-cli/1.16.310 Python/3.8.0 Darwin/18.7.0 botocore/1.13.46
```

## To reproduce:
1. `aws-vault rotate X`

**Comment by mtibben** [comment]
@hunttom can you check if this issue still exists in [v5.1.0](https://github.com/99designs/aws-vault/releases/tag/v5.1.0), and also try with the `--no-session` flag?

**Comment by cread** [comment]
@mtibben I had the same problem when using v5.1.0. Adding `--no-session` fixed it for me...

**Comment by hunttom** [comment]
@mtibben `aws-vault rotate <profile> --no-session` works.

Still two issues:
1. An issue with creating the meta data server `aws-vault exec <profile> -s`
2. Same issue credential rotation `aws-vault rotate <profile>

**Comment by mtibben** [comment]
This was fixed in v5.1.0
